File: src/Spa/PmsClient/src/app/models/PatientAllergy.ts
export class PatientAllergy{
    constructor(      
        public id: number=0,
        public allergy:string="",
        public isfatal:boolean=false,
        public patientid:number=0
     )
    {
        
    }

    get Id() { return this.id };
    set Id(data: number) { this.id = data };

  get Allergy() { return this.allergy };
  set Allergy(data: string) { this.allergy = data };

  get Isfatal() { return this.isfatal };
  set Isfatal(data: boolean) { this.isfatal = data };

  get Patientid() { return this.patientid };
  set Patientid(data: number) { this.patientid = data };
}

File: AuthServer/AuthServer.Infrastructure/Models/PatientAllergy.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace AuthServer.Infrastructure.Models
{
    public partial class PatientAllergy
    {
        public int Id { get; set; }
        public int? PatientId { get; set; }
        public int? AllergyId { get; set; }
        public bool? FatalAllergy { get; set; }
        public DateTime? CreatedOn { get; set; }
        public string CreatedBy { get; set; }
        public string ModifiedBy { get; set; }
        public DateTime? ModifiedOn { get; set; }
        public bool? IsActive { get; set; }

        public virtual AllergyMaster Allergy { get; set; }
        public virtual PatientMaster Patient { get; set; }
    }
}
